src/main/java/CalStringGame/GameManager.java

```diff
@@ -0,0 +1,36 @@
+package CalStringGame;
+
+import expression.Expression;
+import expression.ExpressionBuilder;
+import expression.ExpressionCalculator;
+
+public class GameManager {
+	private static final String TO_STRING = "";
+
+	private String stringExpression;
+	private String result;
+
+	public GameManager() {
+		this.stringExpression = null;
+		this.result = null;
+	}
+
+	public void init(String stringExpression) {
+		this.stringExpression = stringExpression;
+	}
+
+	public void play() {
```

Comment:
요청 파라미터와 응답 값을 가지는 메서드로 리팩토링 해보는건 어떨까요?

```suggestion
	public String play(String stringExpression) {
```

Reply:
적용하였습니다ㅎㅎ